packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-comment-box.tsx

```diff
@@ -415,12 +456,16 @@ export const PullRequestCommentBox = ({
               onDragLeave={handleDragLeave}
               ref={dropZoneRef}
             >
-              <Textarea
+              <PullRequestCommentTextarea
```

Comment:
This newly created component has encapsulated all the user mentions features inside.